### ISSUE TITLE
'jbang edit --live=code' doesn't work on Windows

### ISSUE THREAD

**Issue by OLibutzki** [jbangdev/jbang#244]
Although the `code` binary is in the path, it's not possbible to run VS Code by using `jbang edit --live=code`.

```
jbang --verbose edit --live=code test.java
[jbang] Running code C:\Users\Oliver\.jbang\cache\projects\test.java_jbang_1a6a45ad08c2a567e51a033636e48e3522ce9c5e72150f11525c3fcd4b354df6\test`

java.io.IOException: Cannot run program "code": CreateProcess error=2, Das System kann die angegebene Datei nicht finden
        at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1128)
        at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1071)
        at dev.jbang.cli.Edit.doCall(Edit.java:55)
        at dev.jbang.cli.BaseCommand.call(BaseCommand.java:59)
        at dev.jbang.cli.BaseCommand.call(BaseCommand.java:13)
        at picocli.CommandLine.executeUserObject(CommandLine.java:1933)
        at picocli.CommandLine.access$1100(CommandLine.java:145)
        at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2332)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2326)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2291)
        at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2159)
        at picocli.CommandLine.execute(CommandLine.java:2058)
        at dev.jbang.Main.main(Main.java:14)
Caused by: java.io.IOException: CreateProcess error=2, Das System kann die angegebene Datei nicht finden
        at java.base/java.lang.ProcessImpl.create(Native Method)
        at java.base/java.lang.ProcessImpl.<init>(ProcessImpl.java:478)
        at java.base/java.lang.ProcessImpl.start(ProcessImpl.java:154)
        at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1107)
        ... 12 more
```
Using `jbang edit --live=code.cmd test.java` works. So you might want to adjust the docs accordingly.

**Comment by OLibutzki** [comment]
Hmm, looks like `jbang edit --live=code.cmd test.java` creates a hard link instead of a soft link although I have admin rights.

**Comment by OLibutzki** [comment]
In order to clarify:
```
# jbang edit --live=code hello.java
[jbang] Running `code C:\Users\Oliver\.jbang\cache\projects\hello.java_jbang_277f9643a052b01064bc3bf06316838e3e0ec8e6ea9bd4a4b33aea52905f1447\hello`
[jbang] java.io.IOException: Cannot run program "code": CreateProcess error=2, Das System kann die angegebene Datei nicht finden
[jbang] Run with --verbose for more details
```

```
# jbang edit --live=code.cmd hello.java
[jbang] Running `code.cmd C:\Users\Oliver\.jbang\cache\projects\hello.java_jbang_277f9643a052b01064bc3bf06316838e3e0ec8e6ea9bd4a4b33aea52905f1447\hello`
[jbang] Watching for changes in C:\Entwicklung\jbang
```

**Comment by maxandersen** [comment]
this is fixed now in #271 

`jbang edit --open=code --live hello.java`
